source/_components/sensor.random.markdown

```diff
@@ -11,6 +11,7 @@ logo: home-assistant.png
 ha_category: Utility
 ha_iot_class: "Local Polling"
 ha_release: 0.32
+ha_qa_scale: silver
```

Comment:
I think that we should limit the QA scale to external services/integrations. Things that solely rely on internal code should be excluded.

Reply:
This is just an example.